scripts/redirect_creation/main.tf

```diff
@@ -0,0 +1,52 @@
+
+terraform {
+  backend "remote" {
+    organization = "liquibase"
+    workspaces {
+      name = "liquibase-docs"
+    }
+  }
+}
+
+# use Terraform Remote State Data stanza to read state from prod and staging workspaces
+
+data "terraform_remote_state" "prod" {
+  backend = "remote"
+
+  config = {
+    organization = "liquibase"
+    workspaces = {
+      name = "liquibase-prod"
+    }
+  }
+}
+
+data "terraform_remote_state" "staging" {
+  backend = "remote"
+
+  config = {
+    organization = "liquibase"
+    workspaces = {
+      name = "liquibase-staging"
+    }
+  }
+}
+
+# Need legacy Datical account since s3 buckets for docs are there
+provider "aws" {
+  region     = "us-east-1"
+#  alias      = "datical"
+  access_key = var.datical_access_key
+  secret_key = var.datical_secret_key
+}
+
+# 0 byte objects used for redirects
+resource "aws_s3_object" "redirects" {
```

Comment:
having only this resource in this repo but the other affiliated resources in the infrastructure repo seems weird, but maybe that's normal?

Reply:
Answered here: https://datical.atlassian.net/browse/DAT-12840 with my humble opinion